### PR TITLE
Adding notable technical change for builder sa

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.16

Issue:
N/A

Link to docs preview:
https://78043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-builder-sa

QE review:
- [x] QE has approved this change.

Additional information:

This update came out of what was announced at the 4.16 what's new presentation

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -2411,7 +2419,7 @@ To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However
...
 
-* Previously, role bindings related to the `ImageRegistry`, `Build`, and `DeploymentConfig` capabilities were created in every namespace, even if the capability was disabled. With this release, the role bindings are only created if the cluster capability is enabled on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-34384[*OCPBUGS-34384*])
+* Previously, role bindings related to the `ImageRegistry` and `Build` capabilities were created in every namespace, even if the capability was disabled. With this release, the role bindings are only created if the cluster capability is enabled on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-34384[*OCPBUGS-34384*])
```
Comment (adambkaplan) [inline release_notes/ocp-4-16-release-notes.adoc:2422]: `ImageRegistry` and `Build` are separate capabilities. They are independent of each other with this bug fix.

```suggestion
* Previously, role bindings related to the `ImageRegistry` and `Build` capabilities were created in every namespace, even if the capability was disabled. With this release, the role bindings are only created if the respective cluster capability is enabled on the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-34384[*OCPBUGS-34384*])
```

**Review by adambkaplan — APPROVED** [review]
/lgtm

One minor suggestion, but I won't block merge on it.

/label qe-approved

Marking this as QE approved - we verified these changes pre-merge.

**Comment by bergerhoffer** [comment]
@jitendar-singh can you please QE review this?

Preview: https://78043--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-builder-sa

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Pull request merged**